modules/flowable-ui/flowable-ui-modeler-frontend/src/main/resources/static/modeler/i18n/en.json

```diff
@@ -1,3079 +1,3004 @@
+        },
+        "MAILTASKHTMLVAR": {
+          "TITLE": "HtmlVar",
+          "DESCRIPTION": "The name of a case variable that holds the HTML that is the content of the e-mail."
+        }
+      },
+      "MAILTASKCHARSETPACKAGE": {
+        "MAILTASKCHARSET": {
+          "TITLE": "Charset",
+          "DESCRIPTION": "Allows to change the charset of the email, which is necessary for many non-English languages. "
+        }
+      },
+      "TEXTPACKAGE": {
+        "TEXT": {
+          "TITLE": "TEXT",
+          "DESCRIPTION": "The text of the text annotation."
+        }
+      },
+      "FORMREFERENCEPACKAGE": {
+        "FORMREFERENCE": {
+          "TITLE": "Form reference",
+          "DESCRIPTION": "Reference to a form"
+        }
+      },
+      "DECISIONTASKDECISIONTABLEREFERENCEPACKAGE": {
+        "DECISIONTASKDECISIONTABLEREFERENCE": {
+          "TITLE": "Decision table reference",
+          "DESCRIPTION": "Set the decision table reference"
+        }
+      },
+      "DECISIONTASKDECISIONSERVICEREFERENCEPACKAGE": {
+        "DECISIONTASKDECISIONSERVICEREFERENCE": {
+          "TITLE": "Decision service reference",
+          "DESCRIPTION": "Set the decision service reference"
+        }
+      },
+      "DECISIONTASKTHROWERRORONNOHITSPACKAGE": {
+        "DECISIONTASKTHROWERRORONNOHITS": {
+          "TITLE": "Throw error if no rules were hit",
+          "DESCRIPTION": "Should an error be thrown if no rules of the decision table were hit and consequently no result was found."
+        }
+      },
+      "DECISIONTASKFALLBACKTODEFAULTTENANTPACKAGE": {
+        "DECISIONTASKFALLBACKTODEFAULTTENANT": {
+          "TITLE": "Fallback to default tenant",
+          "DESCRIPTION": "Find decision definition without tenant when previous attempts to find it with tenant failed."
+        }
+      },
+      "HTTPTASKREQUESTMETHODPACKAGE": {
+        "HTTPTASKREQUESTMETHOD": {
+          "TITLE": "Request method",
+          "DESCRIPTION": "Request method (For example - GET,POST,PUT etc)."
+        }
+      },
+      "HTTPTASKREQUESTURLPACKAGE": {
+        "HTTPTASKREQUESTURL": {
+          "TITLE": "Request URL",
+          "DESCRIPTION": "Request URL (For example - http://flowable.org)"
+        }
+      },
+      "HTTPTASKREQUESTHEADERSPACKAGE": {
+        "HTTPTASKREQUESTHEADERS": {
+          "TITLE": "Request headers",
+          "DESCRIPTION": "Line separated HTTP request headers (For example - Content-Type: application/json)."
+        }
+      },
+      "HTTPTASKREQUESTBODYPACKAGE": {
+        "HTTPTASKREQUESTBODY": {
+          "TITLE": "Request body",
+          "DESCRIPTION": "Request body (For example- ${sampleBody})."
+        }
+      },
+      "HTTPTASKREQUESTBODYENCODINGPACKAGE": {
+        "HTTPTASKREQUESTBODYENCODING": {
+          "TITLE": "Request body encoding",
+          "DESCRIPTION": "Request body encoding (For example- UTF-8)."
         }
-    },
-
```

Comment:
Why is the English file included?  The only "diffs" are removing empty lines.

Reply:
That file was caused by my formatting operation

Reply:
Ok, @201405kpb can you make sure that the English file is not included in the PR?